docs/packaging.md

```diff
@@ -0,0 +1,53 @@
+# Application Lifecycle
```

Comment:
Also, should we have this .md file inside the sample/template?  Or under https://github.com/paypal/squbs/tree/RELEASE-0.7.X/docs?  I would prefer the latter.